grabbit/src/main/groovy/com/twcable/grabbit/GrabbitConfiguration.groovy

```diff
@@ -137,11 +138,27 @@ class GrabbitConfiguration {
 
     @CompileStatic
     static class PathConfiguration {
+        private static final String COMMA = ","
+        private static final String SLASH = "/"
+        private static final String EMPTY = ""
         String path
+        Collection<String> excludePaths
         Collection<String> workflowConfigIds
+        
+        Collection<String> modifyExcludePaths(@Nonnull Collection<String> excludePaths)
+        {
+            Collection<String> temp = excludePaths
+            String modifiedExcludePaths = EMPTY;
+            for(String append : temp)
+            {
+                modifiedExcludePaths += this.path+SLASH+append+COMMA;
+            }
+            return modifiedExcludePaths.split(COMMA) as Collection<String>
```

Comment:
One of the reasons I want those unit tests (which in this case wouldn't need JCR mocking), is what would/should happen if this received input like `[null, "foo", "/bar", "./bunny", "\\yikes", "co,ma"]`? (There's nothing preventing a comma being a part of a node's name)